.zuul.yaml

```diff
@@ -57,6 +57,7 @@
         tempest.api.identity.admin.v3.test_credentials.CredentialsTestJSON
         tempest.api.identity.admin.v3.test_tokens.TokensV3TestJSON.test_rescope_token
         tempest.api.identity.admin.v3.test_users.UsersV3TestJSON.test_update_user_password
+      cifmw_test_operator_tempest_expected_failures_list: |
```

Comment:
```suggestion
      cifmw_test_operator_tempest_expected_failures_list: |
        foobar
```

Reply:
Yeah, I was curious if the option can be set empty like this. Do I need to add foobar for it to work? :thinking:

Reply:
You could leave it empty but then it would be better to define the var like this (no need for `|` [1]).

```
cifmw_test_operator_tempest_expected_failures_list: ""
``` 
But I would prefer if you added the `foobar` value. Think about it in terms of code coverage. How much lines of the newly added code to test-operator and tcib would be executed if you lave the value stored in the variable empty in contrast with the `foobar` value? :thinking: [2]

It's definitely good to testy for edge cases like empty value but in this job it would be good if we tried to cover as much code as possible.

[1] [here](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics)
[2] [here](https://github.com/openstack-k8s-operators/test-operator/pull/233/files#diff-19fb6c8a471eb8f571e5e959d274704a582df8fa55a60f500593fcba5efb8823R448)

Reply:
Alright, thank you for your answer. That was an interesting view on this, I did not think about it in that way :smile: